### PR TITLE
Build new /smartstart page

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Add new page `/smartstart`
(confirmed with @SirSamTumless create new page at `/smartstart` and don't add to `/smart-start)`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/smartstart
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check page against [copy doc](https://docs.google.com/document/d/1H8JymT13-pHvPh2KXeXYaAH4HoPYWiwAQXXUDqlGzLM/edit#heading=h.wwhog9r2zrb1)
- Using design similar to [#3178](https://github.com/canonical-web-and-design/web-squad/issues/3178)

## Issue / Card

Fixes [#3367](https://github.com/canonical-web-and-design/web-squad/issues/3367)

## Screenshots

image

### CODE REVIEW

**Comment by anthonydillon** [comment]
> (confirmed with @SirSamTumless create new page at /smartstart and don't add to /smart-start)

Why? does this mean we should move the guides to `smartstart/guide`?

File: templates/smartstart/index.html [hunk 1]
```diff
@@ -0,0 +1,508 @@
+{% extends "smartstart/base_smartstart.html" %}
+
+
+{% block title %}Smart things as a service{% endblock %}
```
Comment (anthonydillon) [inline templates/smartstart/index.html:4]: the title should be "SMART START - Fast-track to market with Ubuntu Core" as per the title of the copy doc.

**Comment by bethcollins92** [comment]
@anthonydillon @SirSamTumless Sam said to me that smart-start/guide will be moved to /smartstart/guide later

File: templates/smartstart/index.html [hunk 2]
```diff
@@ -0,0 +1,508 @@
...
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light is-deep">
+  <div class="row p-divider">
+    <div class="col-4 p-divider__block">
+      <h2>Fast to market</h2>
```
Comment (anthonydillon) [inline templates/smartstart/index.html:44]: Feels like this headings are slightly to large. Think they look better as h4s:
image

File: templates/smartstart/index.html [hunk 3]
```diff
@@ -0,0 +1,508 @@
...
+          width="300",
+          height="300",
+          hi_def=True,
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light is-deep">
```
Comment (anthonydillon) [inline templates/smartstart/index.html:41]: I would probably try this without the `is-deep` to bring more content above the fold.

**Comment by pmahnke** [comment]
it makes no sense to have them in different places @SirSamTumless ???

File: templates/smartstart/index.html [hunk 4]
```diff
@@ -0,0 +1,508 @@
...
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light is-deep">
+  <div class="row">
+    <div class="col-8">
+      <h2>Simple choices</h2>
+      <p>We’ve standardised and accelerated your journey to market</p>
```
Comment (anthonydillon) [inline templates/smartstart/index.html:232]: This sentence should have a full stop

File: templates/smartstart/index.html [hunk 5]
```diff
@@ -0,0 +1,508 @@
...
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">After-sales support</li>
+        <li class="p-list__item is-ticked">Ticket resolution</li>
+        <li class="p-list__item is-ticked">Long term commercial partnership</li>
+      </ul>
+    </div>
+  </div>
+  <div class="u-fixed-width u-align--center">
+    <div>
+      <p class="u-align--center"><i>“Ubuntu is pervasive in business and has a rich ecosystem of partners to accelerate time to market and time to value for organizations that choose to utilize SMART START.”</i></p>
```
Comment (anthonydillon) [inline templates/smartstart/index.html:359]: Please use the Vanilla quote pattern: https://vanillaframework.io/docs/patterns/pull-quote

File: templates/smartstart/index.html [hunk 6]
```diff
@@ -0,0 +1,508 @@
...
+    </ul>
+    <p>
+      <a href="/pricing/devices">Find out more here&nbsp;&rsaquo;</a>
+    </p>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <h2 class="u-sv2">Learn more...</h2>
```
Comment (anthonydillon) [inline templates/smartstart/index.html:431]:
```suggestion
    <h2 class="u-sv2">Learn more&hellip;</h2>
```

**Review by anthonydillon — APPROVED** [review]
Thanks, Beth looking good. Few comments in the code.

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Sorry meant this

**Comment by SirSamTumless** [comment]
Please move /smart-start and smart-start/guide to  /smartstart/ and /smartstart/guide. Could a redirect be set from /smart-start to /smartstart.

File: redirects.yaml [hunk 7]
```diff
@@ -679,6 +679,8 @@ services/server/?: "/server/management"
 shipit.*: "https://blog.canonical.com/2011/04/05/shipit-comes-to-an-end/"
 sitemap.xml?: "/static/files/sitemap.xml"
 sites/.*: "/blog"
+smart-start/?: "/smartstart"
+smart-start/guide/(?P<path>.*)/?: "/smartstart/guide/{path}"
```
Comment (anthonydillon) [inline redirects.yaml:683]: Need another redirect of `/smart-start/guide` to `/smartstart/guide`

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Request for redirect

File: templates/smartstart/index.html [hunk 8]
```diff
@@ -0,0 +1,509 @@
...
+        <li class="p-list__item is-ticked">Bring your IoT devices to market in the blink of an eye with SMART START</li>
+        <li class="p-list__item is-ticked">Focus on your apps, we handle the rest</li>
+        <li class="p-list__item is-ticked">Fast to market, low risk, and scalable</li>
+      </ul>
+      <p>
+        Canonical offers full-service enablement, customisation and development to get your first device to market. App store and security updates guaranteed.
+      </p>
+      <p>
+        <a class="p-button--positive" href="/pricing/devices">Pricing</a>
+        <a href="/smart-start/guide">A guide to SMART START&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/smartstart/index.html:24]: Should be `/smartstart/guide` now

Comment (bethcollins92) [reply templates/smartstart/index.html:24]: ahh yes, forgot to change them on that page. I'll do that now

**Comment by anthonydillon** [comment]
Should there be navigation. There is no clear way to navigate back smartstart from the guide.

**Comment by anthonydillon** [comment]
The dynamic contact us for needs to have the return URL updated.

File: navigation.yaml [hunk 9]
```diff
@@ -182,6 +182,24 @@ raspberrypi:
...
 
+smartstart:
+  title: SMART START
+  path: /smartstart
+
+  children:
+    - title: Overview
+      path: /smartstart
+    - title: Guide
+      path: /smartstart/guide
```
Comment (anthonydillon) [inline navigation.yaml:193]:
```suggestion
    - title: Guide
      path: /smartstart/guide
      persist: True
```

**Review by anthonydillon — APPROVED** [review]
Looks good with one small suggestion to keep the navigation across all guide pages.

**Comment by meralonso** [comment]
@bethcollins92 can you put this quote on a separate section and use white background? And please invert background colours on sections underneath.

Screenshot 2020-11-02 at 11 57 50

**Comment by bethcollins92** [comment]
@meralonso Making a new PR to address now, thanks

**Pull request merged**